openreview/openreview.py

```diff
@@ -475,6 +476,30 @@ def post_profile(self, profile):
         response = self.__handle_response(response)
         return Profile.from_json(response.json())
 
+    def merge_profiles(self, profileTo, profileFrom):
+        """
+        Merges two Profiles
+
+        :param profileTo: Profile object to merge to
+        :type profileTo: Profile
+        :parm profileFrom: Profile object to merge from
```

Comment:
looks good. my one suggestion is that we add just a little more description here: maybe something like "Profile to merge from (this profile will be deleted)".

Reply:
Fixed